tests/Unit/Domain/Structure/Test_OrientationMapHelpers.cpp

```diff
@@ -123,22 +123,25 @@ void test_2d_with_orientation(
       Affine2D{Affine(-1.0, 1.0, 2.3, 4.5), Affine(-1.0, 1.0, 0.8, 3.1)};
   const auto map =
       domain::make_coordinate_map<Frame::Logical, Frame::Inertial>(affine);
-  const auto map_oriented =
-      domain::make_coordinate_map<Frame::Logical, Frame::Inertial>(
-          domain::CoordinateMaps::DiscreteRotation<2>{orientation_map}, affine);
+  const auto logical_coords = logical_coordinates(mesh);
 
   Variables<tmpl::list<ScalarTensor, Coords<2>>> vars(extents.product());
   // Fill ScalarTensor with x-coordinate values
-  get(get<ScalarTensor>(vars)) = get<0>(map(logical_coordinates(mesh)));
+  get(get<ScalarTensor>(vars)) = get<0>(map(logical_coords));
   get<Coords<2>>(vars) = map(logical_coordinates(mesh));
   const auto oriented_vars = orient_variables(vars, extents, orientation_map);
 
   Variables<tmpl::list<ScalarTensor, Coords<2>>> expected_vars(
       extents.product());
-  get(get<ScalarTensor>(expected_vars)) = get<0>(
-      map_oriented(logical_coordinates(orientation_map.inverse_map()(mesh))));
-  get<Coords<2>>(expected_vars) =
-      map_oriented(logical_coordinates(orientation_map.inverse_map()(mesh)));
+  const std::array<DataVector, 2> oriented_logical_coords = discrete_rotation(
+      orientation_map,
+      std::array{logical_coordinates(orientation_map.inverse_map()(mesh))[0],
+                 logical_coordinates(orientation_map.inverse_map()(mesh))[1]});
```

Comment:
For my understanding: don't we prefer the `get` interface for tensor components when accessing physical components, and the `[]` interface when accessing stored data? I thought the intent of the `get` interface was to be unambiguous that you are accessing a component vs a storage entry.

Reply:
Yea, you're right. Some of these things are `std::array`s, but not these. Thanks!